web/app/view/ReportConfigController.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ * Copyright 2016 Andrey Kunitsyn (abyss@fox5.ru)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.view.ReportConfigController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.reportConfigDialog',
+
+    init: function () {
+        var store = this.lookupReference('eventTypeField').getStore();
+        if (store.getCount() === 0) {
+            store.add({
+                type: Traccar.store.ReportEventTypes.allEvents,
```

Comment:
Make sure that you always import all the types that you use. It might not give any errors at this point because it's probably imported somewhere already, but it might fail at some point in future or it might result in some issues after minification. Applies to everything, not just this line.